general/management/commands/gm_worker_async_tasks.py

```diff
@@ -90,3 +92,17 @@ def task_whitelist_user(self, gearman_worker, gearman_job):
                 count_done = count_done + 1
             self.write_stdout("Finished processing one ticket, %d remaining" % (len(tickets)-count_done))
         return 'true' if len(tickets) == count_done else 'false'
+
+    def task_email_random_sound(self, gearman_worker, gearman_job):
+        self.write_stdout("Notifying user of random sound of the day")
+        random_sound_id = gearman_job.data
+        random_sound = Sound.objects.get(id=random_sound_id)
+
+        send_mail_template(u'Random sound of the day.',
```

Comment:
don't need a . in the subject. Maybe something a little more exciting?  "One of your sounds has been chosen as random sound of the day!"